src/endpoint/s3/ops/s3_get_object.js

```diff
@@ -70,7 +70,10 @@ async function get_object(req, res) {
         }
         throw err;
     }
-
+    if (params.object_md.size < 4096) {
```

Comment:
can you move this number to config.js?
also use it in namespace_s3 where you send the range.

Reply:
This condition is also not enough for all our other namespaces to work.
I think you need to change to something like:
```suggestion
    if (params.object_md.first_range_data && params.object_md.first_range_data.length >= params.object_md.size) {
```